app/uk/gov/hmrc/merchandiseinbaggagefrontend/config/AppConfig.scala

```diff
@@ -33,12 +33,24 @@ class AppConfig() extends MongoConfiguration {
   val contactUrl = s"$contactHost/contact/contact-hmrc-unauthenticated?service=$serviceIdentifier"
 }
 
+object AppConfigSource {
+  val configSource: String => ConfigSource = ConfigSource.default.at
+}
+
 trait MongoConfiguration {
   lazy val mongoConf: MongoConf = configSource("mongodb").loadOrThrow[MongoConf]
 }
 
 final case class MongoConf(uri: String, host: String = "localhost", port: Int = 27017, collectionName: String = "declaration")
 
-object AppConfigSource {
-  val configSource: String => ConfigSource = ConfigSource.default.at
-}
\ No newline at end of file
+trait MibConfiguration {
+  lazy val mibConf: MIBConf = configSource("microservice.services.merchandise-in-baggage").loadOrThrow[MIBConf]
+  import mibConf._
+  lazy val mibPort: Int = port
+  lazy val baseUrl: String = s"$protocol://$host:$mibPort"
+  lazy val serviceUrl: String = s"/merchandise-in-baggage"
+  lazy val declarationsUrl: String = s"$serviceUrl/declarations"
+  lazy val persistDeclarationsUrl: String = s"$baseUrl$declarationsUrl"
```

Comment:
these 3 seem to be used by the connector only. Maybe move them there consistent with the other connectors? That would be easier to read.

Reply:
agree looks a bit messy but again is to avoid strings in tests - wire mock is not asserting on absolute paths so `declarationsUrl` is used for wire mock assertion